pages/lists/[listId]/index.tsx

```diff
@@ -1,5 +1,18 @@
-const ListsHub = () => {
-  return <div>Redirect to Lists Overview</div>;
+import { GetServerSidePropsContext } from "next";
+
+const ListIndex = () => {
+  return <></>;
```

Comment:
nit: return null

```suggestion
  return null;
```

Reply:
When do you return a fragment? Just for my own knowledge as I've seen errors when returning null

Reply:
You only ever need fragments if it's not a single JSX node you're returning.

e.g.

✅ Good

return <>
	<p>paragraph one</p>
	<p>paragraph one</p>
</>

❌ compilation error

return <p>paragraph one</p><p>paragraph one</p>

Whenever you need to return nothing, `null` seems to be what people have settled on.